### PR TITLE
[17.0][MIG] crm_phonecall: Migration to 17.0

### PR DESCRIPTION
**Pull request by peluko00**
Module migrated to version 17.0

cc https://github.com/APSL 149031

@miquelalzanillas @lbarry-apsl @javierobcn @mpascuall please review

Issue https://github.com/OCA/crm/issues/525

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Now you get it! Please check the rest of your pull requests with the same problem. Another thing: you don't need to close existing pull request. Just fix it locally with the same branch name, and then forced push the result to your remote.

/ocabot migration crm_phonecall

**Comment by peluko00** [comment]
> Now you get it! Please check the rest of your pull requests with the same problem. Another thing: you don't need to close existing pull request. Just fix it locally with the same branch name, and then forced push the result to your remote.
> 
> /ocabot migration crm_phonecall

OK! Sorry for the mistakes

**Comment by peluko00** [comment]
Module migrated to version 17.0

cc https://github.com/APSL 149031

@miquelalzanillas @lbarry-apsl @javierobcn @samoros1 please review

Issue https://github.com/OCA/crm/issues/525

File: crm_phonecall/views/crm_phonecall_view.xml [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+                    icon="fa-hand-pointer-o"
+                    name="action_button_convert2opportunity"
+                    string="Convert to Opportunity"
+                    type="object"
+                />
+                <field name="partner_phone" />
+                <field name="user_id" />
+                <field invisible="1" name="tag_ids" />
+                <field invisible="1" name="create_date" />
+                <field invisible="1" name="opportunity_id" />
```
Comment (CRogos) [inline crm_phonecall/views/crm_phonecall_view.xml]: invisible -> column_invisible needs to be changed as well.
Also in other views.

**Review by CRogos — CHANGES_REQUESTED** [review]
Could you cleanup the commit history according to: https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

remove commits by oca-git-bot, ...

File: crm_phonecall/views/crm_phonecall_view.xml (hunk 1 above)
Comment (peluko00) [reply crm_phonecall/views/crm_phonecall_view.xml]: Fix it! Please review @CRogos

Comment (CRogos) [reply crm_phonecall/views/crm_phonecall_view.xml]: Could you cleanup the commits as well?
The code changes LGTM, but did not test the addon in detail.

**Review by javierobcn — APPROVED** [review]
Works well in runboat

**Review by lbarry-apsl — APPROVED** [review]
Works, tested on Runboat

**Comment by lbarry-apsl** [comment]
I was hoping to see the categories in the logged calls, it could be an improvement.

File: crm_phonecall/views/crm_phonecall_view.xml (hunk 1 above)
Comment (peluko00) [reply crm_phonecall/views/crm_phonecall_view.xml]: Done! Can you review again please @CRogos

**Comment by carolinafernandez-tecnativa** [comment]
Hi could you please rebase and squash administrative commits? Thanks!

**Comment by peluko00** [comment]
> Hi could you please rebase and squash administrative commits? Thanks!

Done! Ready for review

**Comment by CRogos** [comment]
There are still a lot of commits which should be squashed. Here some examples, but there are a lot more: 

image
https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

**Review by pedrobaeza — APPROVED** [review]
Although technically not all the administrative are squashed, let's move with this as this is blocking other migrations and we can continue the squashing on the next migration.

/ocabot merge nobump

**Pull request merged**